modules/mailer/mailer.go

```diff
@@ -38,7 +38,7 @@ func NewMessageFrom(to []string, fromDisplayName, fromAddress, subject, body str
 	msg := gomail.NewMessage()
 	msg.SetAddressHeader("From", fromAddress, fromDisplayName)
 	msg.SetHeader("To", to...)
-	msg.SetHeader("Subject", subject)
+	msg.SetHeader("Subject", setting.MailService.SubjectPrefix+subject)
```

Comment:
Can you add a space between prefix and subject if prefix is set?

Reply:
OK I've changed that. This marks a change from the behaviour in Gogs.